### PR TITLE
(+) #CTE-8 #CTE-9 #CTE-12 #CTE-13 Added options for quick format such as Remove duplicate lines, Remove blank lines and Trim whitespaces

### PR DESCRIPTION
**Pull request by alexfdezsauco**
Fixes #CTE-8, #CTE-9 or #CTE-12 (duplicate issue), #CTE-13.

### Checklist

- [x] I am listed in the CONTRIBUTORS file
- [X] I have cleaned up the commit history (use rebase and squash)

### Changes proposed in this pull request:
Added options for quick format such as Remove duplicate lines, Remove blank lines and Trim whitespaces

### CODE REVIEW

File: src/CsvTextEditor/Commands/EditRemoveBlankLinesCommandContainer.cs [hunk 1]
```diff
@@ -0,0 +1,27 @@
+﻿namespace CsvTextEditor
```
Comment (GeertvanHorrik) [inline src/CsvTextEditor/Commands/EditRemoveBlankLinesCommandContainer.cs]: file header

File: src/CsvTextEditor/Commands/EditRemoveDuplicateLinesCommandContainer.cs [hunk 2]
```diff
@@ -0,0 +1,27 @@
+namespace CsvTextEditor
```
Comment (GeertvanHorrik) [inline src/CsvTextEditor/Commands/EditRemoveDuplicateLinesCommandContainer.cs]: header

File: src/CsvTextEditor/Commands/EditTrimWhitespacesCommandContainer.cs [hunk 3]
```diff
@@ -0,0 +1,27 @@
+namespace CsvTextEditor
```
Comment (GeertvanHorrik) [inline src/CsvTextEditor/Commands/EditTrimWhitespacesCommandContainer.cs]: header

File: src/CsvTextEditor/Constants.cs [hunk 4]
```diff
@@ -78,6 +78,10 @@ public static class Edit
 
             public const string DeleteLine = "Edit.DeleteLine";
             public static readonly InputGesture DeleteLineInputGesture = new InputGesture(Key.L, ModifierKeys.Control);
+            
+            public const string RemoveBlankLines = "Edit.RemoveBlankLines";
```
Comment (GeertvanHorrik) [inline src/CsvTextEditor/Constants.cs]: also add the default input gestures (but set as null if not needed). this makes it easier for us to change this in the future.

File: src/Orc.CsvTextEditor/Services/CsvTextEditorService.cs [hunk 5]
```diff
@@ -100,6 +100,46 @@ public CsvTextEditorService(TextEditor textEditor, ICommandManager commandManage
         public event EventHandler<CaretTextLocationChangedEventArgs> CaretTextLocationChanged;
         public event EventHandler<EventArgs> TextChanged;
 
+        public void RemoveBlankLines()
+        {
+            var document = _textEditor.Document;
```
Comment (GeertvanHorrik) [inline src/Orc.CsvTextEditor/Services/CsvTextEditorService.cs]: Please add debug logging for this like this:

```
Log.Debug("Removing blank lines");
```

File: src/Orc.CsvTextEditor/Services/CsvTextEditorService.cs [hunk 6]
```diff
@@ -100,6 +100,46 @@ public CsvTextEditorService(TextEditor textEditor, ICommandManager commandManage
...
+                    document.Remove(documentLine.Offset, documentLine.TotalLength);
+                }
+            }
+
+            UpdateText(document.Text);
+        }
+
+        public void TrimWhitespaces()
+        {
+            StringBuilder builder = new StringBuilder();
```
Comment (GeertvanHorrik) [inline src/Orc.CsvTextEditor/Services/CsvTextEditorService.cs]: Please add debug logging for this like this:

```
Log.Debug("Trimming white spaces");
```

**Comment by alexfdezsauco** [comment]
Fixed

**Pull request merged**